### PR TITLE
chore: update native library versions

### PR DESCRIPTION
**Pull request by charliecruzan-stripe**
# Motivation
Keeping up-to-date with stripe-react-native

# Summary
iOS to 23.10.0
Android to 20.27.0

### CODE REVIEW

File: stripe-identity-react-native.podspec [hunk 1]
```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 # When stripe_version is updated, also need to update stripe_version in https://github.com/stripe/stripe-react-native/blob/master/stripe-react-native.podspec
-stripe_version = '~> 23.8.0'
+stripe_version = '~> 23.10.0'
```
Comment (ccen-stripe) [inline stripe-identity-react-native.podspec:6]: Seems we should update this to `23.11.0`
https://github.com/stripe/stripe-ios/releases/tag/23.11.0

Comment (charliecruzan-stripe) [reply stripe-identity-react-native.podspec:6]: if we do that we won't have a corresponding version for 0.29.0 of stripe-react-native: https://github.com/stripe/stripe-react-native/blob/master/stripe-react-native.podspec#L5 (we had to run a release already)

Comment (ccen-stripe) [reply stripe-identity-react-native.podspec:6]: ack, could you merge this and create a PR with `23.11.0`

**Review by ccen-stripe — APPROVED** [review]

File: stripe-identity-react-native.podspec (hunk 1 above)
Comment (ccen-stripe) [reply stripe-identity-react-native.podspec:6]: as discussed offline, will drop this PR and release with the latest native versions in https://github.com/stripe/stripe-identity-react-native/pull/131

**Comment by ccen-stripe** [comment]
as discussed offline, will drop this PR and release with the latest native versions in https://github.com/stripe/stripe-identity-react-native/pull/131

**Pull request closed without merging**